wire-ios-request-strategy/Sources/Message Sending/MessageInfoExtractorTests.swift

```diff
@@ -231,6 +249,38 @@ final class MessageInfoExtractorTests: XCTestCase {
         XCTAssertEqual(messageInfo.listClients, expectedListClients)
     }
 
+    // MARK: - Helpers
+    
+    private func internalTest_NativePush(expectedNativePush: Bool, message: GenericMessage, file: StaticString = #file, line: UInt = #line) async throws {
+        var conversation: ZMConversation!
+
+        let conversationID = try await context.perform { [self] in
+            _ = modelHelper.createSelfUser(id: Scaffolding.selfUserID.uuid,
+                                               domain: Scaffolding.selfUserID.domain,
+                                               in: context)
+            let selfClient = modelHelper.createSelfClient(id: Scaffolding.selfClientID, in: context)
+
+            let userA = modelHelper.createUser(qualifiedID: Scaffolding.userAID, in: context)
+            _ = modelHelper.createClient(id: Scaffolding.clientAID, for: userA)
+
+            let userB = modelHelper.createUser(in: context)
+            _ = modelHelper.createClient(for: userB)
+
+            conversation = ZMConversation.insertGroupConversation(moc: context, participants: [userA, selfClient.user!, userB])
+            conversation.remoteIdentifier = Scaffolding.conversationID.uuid
+            conversation.domain = Scaffolding.conversationID.domain
+            mockProteusMessage.conversation = conversation
+            return try XCTUnwrap(conversation.qualifiedID)
```

Comment:
Consider passing `file` and `line` to this function.

Reply:
done